### ISSUE TITLE
Feature request: skip standard packages

### ISSUE THREAD

**Issue by rutigl** [go101/golds#44]
golds has no option to create help for selected package only (not including standard packages).
Could you please think of adding such an option?
I mean, index.html links only package itself, not all imports, and of course the html files are generated for package only.

**Comment by zigo101** [comment]
the closest option might be `-wdpkgs-listing=solo`. With this option, docs of dependencies modules (including std packages) will still be generated but not listed in the index page.

**Comment by rutigl** [comment]
Yes, thank you, at least it shows only desired package in index.html.

**Comment by zigo101** [comment]
A similar old issue: https://github.com/go101/golds/issues/9

Now, the docs of a depended module/package also depend on the modules/packages depend on it.
This can be avoided by providing less information during docs generation, which needs much effort.
I will implement it when I get enough spare time.
